exonum/src/node/mod.rs

```diff
@@ -437,20 +436,6 @@ pub struct ConnectListConfig {
 }
 
 impl ConnectListConfig {
-    /// Creates `ConnectListConfig` from validators public configs.
-    pub fn from_node_config(list: &[NodePublicConfig], node: &NodePrivateConfig) -> Self {
```

Comment:
This method was copied to `exonum-parameters`. I feel like it wasn't in a right place, invlolving weird modules coupling.